relationships/synthesis/APM_CALLS_DATABASE.yml

```diff
@@ -0,0 +1,20 @@
+relationships:
```

Comment:
can we name the files `APM-APPLICATION-to-DATABASE` & `EXT-SERVICE-to-EXT-PIXIE_DNS`? 

That goes a bit more in line with the proposal we did.
Main changes:
- `to` instead of the relationship type, so a file can contain multiple types of relationships
- USE `-` instead of `_` since we don't use `-` in the domain or type names